test/python/fx_importer/sparse_test.py

```diff
@@ -158,7 +158,14 @@ def sparse_jit(f, *args, **kwargs):
     for a in args:
         if a.layout is torch.sparse_coo:
             xargs.append(a.values().numpy())
-            xargs.append(a.indices().numpy())
+            # Construct the additional position array required by MLIR with data
+            # array([0, nnz]).
+            nnz = a.indices().shape[1]
```

Comment:
I have been puzzled why torch.sparse does not just expose the nnz() as a method :-)

Reply:
actually _nnz() seems to work ;-)